pages/index.js

```diff
@@ -1,7 +1,7 @@
 import TopicBox from "../components/molecules/TopicBox";
 import Layout from "../components/layout";
 import { getTopics, getLocalData } from "./api/topics";
-import { useContext } from "react";
+import React, { useContext, useRef } from "react";
```

Comment:
you can add useEffect here instead of React

Reply:
Good suggestion, will do.